src/bindings/python/requirements_test.txt

```diff
@@ -21,7 +21,7 @@ flake8-rst-docstrings
 pygments
 flake8-string-format
 flake8-variables-names
-flake8_builtins<2.2.0 # Issue 124276
+flake8_builtins
```

Comment:
```suggestion
flake8_builtins<2.3.0
```

Reply:
propose to set upper-bound to avoid backward-compatible issues some day.
dependant-bot will care about further updating this upper-bound if any,

Reply:
Agree, newest version already pop out with some problems, that's why Alina made this limitation. Let's keep it for now, test and make further decision later.